API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/feature-flags.json

```diff
@@ -0,0 +1,17 @@
+{
+  "request": {
+    "method": "GET",
+    "urlPath": "/wpcom/v2/mobile/feature-flags",
+    "queryParameters": {
+      "_locale": {
+        "matches": "(.*)"
+      }
+    }
+  },
+  "response": {
+    "status": 200,
+    "jsonBody": {
+      "dashboard_card_domain": true
```

Comment:
Mocking this response causes the card to be shown for eligible sites. Note that this is an app-specific request, not a website-specific. It's sent on the app launch.

Reply:
question: what would happen if the feature flag is removed? should we ensure that in the PR that removes the feature flag (not sure if it's removed via an app code change or not since it looks like it's server side) this mock file/line to turn on feature flag is removed too? wdyt?

Reply:
I assume that the feature flag is removed in the app (but @staskus knows better for sure), and once this happens, this mock file won't be necessary any longer. I'm not sure if having it will harm, though. The same `/wpcom/v2/mobile/feature-flags` request matcher might be useful for testing the future feature flags, so we'd probably remove just the line.